builds/misc/packages-release.yaml

```diff
@@ -435,5 +435,161 @@ stages:
             BASE_VERSION=`cat $BUILD_SOURCESDIRECTORY/edgelet/version.txt`
             $(Build.SourcesDirectory)/scripts/linux/publishReleasePackages.sh -p $(os) -w $(System.ArtifactsDirectory) -d $(System.ArtifactsDirectory)/$(artifactName) -v "$BASE_VERSION" -s "github.com" -b $(Build.SourceBranch)
           env:
-            GITHUB_PAT: "$(GitHubAccessToken)"    
-          name: publish_artifacts 
+            GITHUB_PAT: "$(GitHubAccessToken)"
+          name: publish_artifacts
+
+##############################################################################
+  - stage: smokeTests
+##############################################################################
+    displayName: Post-release Smoke Tests
+    dependsOn:
+      - PublishPackagesMicrosoft
+      - PublishPackagesGithub
+    jobs:
+    - job: smokeTests
+      displayName: Release Artifact Smoke Tests
+      strategy:
+        matrix:
+          Ubuntu1804-amd64:
+            pool_name: 'Azure-IoT-Edge-1ES-Hosted-Linux'
+            agent_demands: 'ImageOverride -equals agent-aziotedge-ubuntu-18.04-docker'
+            os: ubuntu18.04
+            arch: amd64
+            ext: deb
+          Ubuntu1804-arm64:
+            pool_name: 'Azure-IoT-Edge-1ES-Hosted-Linux-Arm64'
+            agent_demands: 'ImageOverride -equals agent-aziotedge-ubuntu-18.04-arm64'
+            os: ubuntu18.04
+            arch: arm64
+            ext: deb
+
+      pool:
+        name: $(pool_name)
+        demands:
+        - $(agent_demands)
+
+      steps:
+      - checkout: azure-iotedge
+        submodules: recursive
+      - checkout: self
+      - task: AzureKeyVault@1
+        displayName: Get secrets
+        inputs:
+          azureSubscription: $(az.subscription)
+          keyVaultName: $(kv.name)
+          secretsFilter: >-
+            IotEdge1-PAT-msazure
+      - task: Bash@3
+        displayName: Setup Test Agent
+        condition: true
```

Comment:
This hard-coded condition seems unnecessary

Reply:
I agree!